Libraries/test/TestPowerShellFunctions/PowerShellScriptsAsFunctions/PowerShellScriptsAsFunctions.csproj

```diff
@@ -21,12 +21,9 @@
     </Content>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Amazon.Lambda.Core" Version="1.0.0" />
-    <PackageReference Include="Amazon.Lambda.Serialization.Json" Version="1.3.0" />
-    <PackageReference Include="Amazon.Lambda.TestUtilities" Version="1.0.0" />
-  </ItemGroup>
-  <ItemGroup>
-    <DotNetCliToolReference Include="Amazon.Lambda.Tools" Version="2.2.0" />
```

Comment:
Reason Amazon.Lambda.Tools is removed? Not used anymore?

Reply:
Yes, it was never meant to be on this test project and it was using the old style of .NET tools before .NET Core Global Tools existed.